skrub/_drop_null.py

```diff
@@ -0,0 +1,54 @@
+# drop columns that contain all null values
```

Comment:
nitpick: can we have the same name for the class, the module and the test module? eg `DropNullColumns`, `drop_null_columns`, `test_drop_null_columns` (or the singular version if you prefer)